.github/ISSUE_TEMPLATE/2_feature_query.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Query Feature Request
+about: Use this template for requesting a query within a user story.
+labels: query, enhancement, needs estimate
+title: 🟩 <Query Name>
+---
+
+# Description
+<!-- A clear and concise description of the feature. -->
+
+## Stakeholders
+<!-- The main points of contact for questions relating to the scope of the feature. -->
+| Product   | Engineering |
+| --------- | ----------- |
+| @Jane     | @John       |
+
+## Model
+<!-- A screenshot or reference to the slice of the model in context. -->
+
+## Engineering Requirements
+<!-- List of engineering items required as part of the feature. -->
+
+1. **Authorization**
+   - Who's authorized to execute this query? Roles, Groups?
+2. **Schemas** - Use [zod](http://zod.dev)
+   - Input/Output schemas
+3. **Routing**
+   - Define REST query paths (GET) - params, query string
+   - Include pagination requirements
+4. **Query**
+   - Define repository interface used to query this projection
+   - New interface might be required
```

Comment:
In most cases we don't yet use a repo pattern so this would be a description of the SQL/sequelize query?

Reply:
Yep, that's the idea. We can clarify the situation if you want